src/cli/templates/create/package.json

```diff
@@ -9,36 +9,63 @@
     "webBundle": "bin/web.bundle.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "watch": "node webpack.js --watch",
-    "compile": "node webpack.js --compile"
+    "test": "npm run lint && npm run mocha",
+    "lint": "eslint ./*.js src/",
```

Comment:
should we also include `jsx`? even though it's discouraged and I think we will eventually stop using this extension it's still nice to support in linting should it happen to occur

Reply:
We should also skip the bundled files, right? `.eslintignore` can be used for it

Reply:
If we use `standard` instead of plain `eslint`, it ignores the correct files automatically. Maybe `prettier` does too?

Reply:
I had to configure ignore for the botpress project itself to ignore bundles.
I'd go with prettier as it's already used in botpress, seems logical to promote the same code style for bots.